rfc6962-bis.xml

```diff
@@ -351,18 +351,30 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
+            When creating a precertificate, the CA MAY omit the <xref target="RFC5280">subjectAltName</xref> extension. Instead, the CA MUST include a <xref target="redacted_san_extension">Redacted SAN</xref> extension, containing the entries that will be included in the certificate's subjectAltName extension, in a redacted form.
           </t>
           <t>
-            Wildcard <spanx style="verb">*</spanx> labels MUST NOT be redacted. However, if the complete leftmost label of a DNS-ID or CN-ID is <spanx style="verb">*</spanx>, it is considered redacted for the purposes of determining if the label to the right may be redacted. For example, if a certificate contains a DNS-ID of <spanx style="verb">*.top.secret.example.com</spanx>, then the corresponding DNS-ID in the precertificate could contain <spanx style="verb">*.?.?.example.com</spanx> instead, but not <spanx style="verb">?.?.?.example.com</spanx> instead.
+            One or more labels in each <xref target="RFC6125">DNS-ID</xref> can be replaced with <spanx style="verb">REDACTED(label)</spanx> labels.
+          </t>
+          <t>
+            The redacted label is defined as follows:
+            <vspace blankLines='1' />
+            REDACTED(label) = ? || HEX(HASH(serial_number || label))
+            Where || denotes concatenation, HASH is the hash function from the TBSCertificate's signature field (<xref target="RFC5280">RFC5280, Section 4.1.2.3.</xref>), serial_number is the precertificate's serial number, consisting of the ASN.1 INTEGER tag, length and content bytes.
+          </t>
+          <t>
+            Wildcard <spanx style="verb">*</spanx> labels MUST NOT be redacted. However, if the complete leftmost label of a DNS-ID or CN-ID is <spanx style="verb">*</spanx>, it is considered redacted for the purposes of determining if the label to the right may be redacted. For example, if a certificate contains a DNS-ID of <spanx style="verb">*.top.secret.example.com</spanx>, then the corresponding DNS-ID in the precertificate could contain <spanx style="verb">*.REDACTED(top).REDACTED(secret).example.com</spanx> instead, but not <spanx style="verb">REDACTED(*).REDACTED(top).REDACTED(secret).example.com</spanx> instead.
           </t>
         </section>
-        <section title="Redacted Labels Certificate Extension" anchor="redaction_extension">
+        <section title="Redacted SAN Certificate Extension" anchor="redacted_san_extension">
+          <t>
+            The Redacted SAN extension is a non-critical extension (OID 1.3.101.77) identical in structure to the subjectAltName extension, the difference being that dNSName entries MAY contain redacted labels. The purpose of this extension is to inform TLS clients of the dNSNames that were redacted and the degree of redaction, while minimizing the complexity of TBSCertificate reconstruction, as described in <xref target="reconstructing_tbscertificate"/>.
```

Comment:
@eranmes Given that Symantec have implemented the redaction mechanism in -16, perhaps we shouldn't "reuse" the 1.3.101.77 OID for the new Redacted SAN extension.  (It would be a shame to lose that particular OID though: it's your easter egg ;-) ).